### DISCUSSION TITLE
Can I re-authorize OAuth Apps after revoking its access?

### DISCUSSION THREAD

**Discussion by magalhaesdavi** [community/community#23687] [New to GitHub]
<p>If I revoke VSCode access to my account, can I re-authorize it later? I was goint to revoke it but then this message showed up “GitHub for VSCode will no longer be able to access your GitHub account. You cannot undo this action.” and I got confused.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>I’ve definitely done that. Not with VSCode, but with other OAuth apps. What you cannot undo is revoking that token, but you can do a fresh authorization afterwards (assuming the app doesn’t have a bug that makes it get stuck on the revoked token).</p>
